.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+* [ ] Have you followed our guidelines in CONTRIBUTING.md? 
+* [ ] Have you self-reviewed your code and corrected any misspellings? 
+* [ ] Have you written documentation that is easy to understand?
+* [ ] Have you written descriptive commit messages? 
+* [ ] Have you added NumPy docstrings for newly added functions? 
+* [ ] Have you formatted your code using black? 
+* [ ] If you have introduced a new functionality, have you added adequate unit tests?
+* [ ] Have all tests passed in your local clone? 
+* [ ] If you have introduced a new functionality, have you added an example notebook?
+* [ ] Have you kept your pull request small and limited so that it is easy to review? 
+* [ ] Have any dependent changes been merged in downstream modules of this branch? 
```

Comment:
I'm not sure what this means. Does it mean that your changes don't break other code?

Reply:
Sorry, I should specify that my question here is specifically on line 11.

Reply:
I guess what I meant here was to avoid merge conflicts by making sure to pull the latest changes on that branch etc. Even if git automatically manages this, maybe it is good to encourage that users think about it before submitting a PR?  I dont know - we can either re-formulate this or I am also happy to just remove this point.

Reply:
Maybe "Have the newest changes from this branch been merged?". I suppose, as you mention, that it isn't really required, but it is probably good practice to avoid unforeseen issues (and conflicts that git doesn't detect).

Reply:
Sounds good, I have changed it according to your suggestion. Besides good practice it could also be helpful for the reviewer to be aware of unmerged changes, in case this bullet point is not checked.